prism/src/main/java/org/apache/falcon/resource/AbstractExtensionManager.java

```diff
@@ -266,4 +270,17 @@ protected static void checkIfExtensionJobNameExists(String jobName, String exten
                     Response.Status.INTERNAL_SERVER_ERROR);
         }
     }
+
+    private JSONObject getEntitiesStatus(List<String> entities, EntityType type) throws JSONException, FalconException {
+        JSONObject entityObject = new JSONObject();
+        for (String entity : entities) {
+            try {
+                EntityUtil.getEntity(type, entity);
+            } catch (EntityNotRegisteredException e) {
+                entityObject.put(entity, ENTITY_NOT_EXISTS_STATUS);
```

Comment:
By JSON standards, the keys are usually constant strings. This should be `{ "name" : "abc, "status" : "EXISTS"}`

This makes it easy to consume programmatically also.